src/clarity-angular/datagrid/datagrid-filter.spec.ts

```diff
@@ -34,22 +34,22 @@ export default function(): void {
                 expect(component.open).toBe(true);
             });
 
-            it("registers to the Filters provider", function() {
+            it("registers to the FiltersProvider provider", function() {
                 expect(filterService.getActiveFilters()).toEqual([]);
-                component.filter = filter;
+                component.setFilter( filter );
```

Comment:
Please use `component.customFilter = filter` everywhere in this spec, since that's what the `@Input` does. You jumped a step there, not a big deal right now but the setter with the `@Input` might do more in the future.